src/lib/CofThy.mli

```diff
@@ -12,6 +12,7 @@ val consistency : disj_thy -> [`Consistent | `Inconsistent]
     then it does so immediately. Otherwise, it is held "on deck" and repeatedly decomposed when testing
     sequents. The function [consistency] will consider cofibrations on deck. *)
 val assume : disj_thy -> cof -> disj_thy
+val assumes : disj_thy -> cof list -> disj_thy
```

Comment:
This name is a little grammatically awkward 😜  I am not sure I have a better name for it, but do we need it in the public interface?

Maybe `assume_list` would be a little better.

Reply:
LOL let me re-refactor. Still we should change the private name.

Reply:
Okay I took a radical approach to just replace the old interface. LOL

We do not need them, but `assume` naturally takes a list.